log/logClient.go

```diff
@@ -491,3 +508,39 @@ func (fd *Feeder) DestroyClient() error {
 
 	return nil
 }
+
+func selectLabelsAlerts(o Options, res *pb.Alert) error {
+
+	if len(o.Selector) <= 1 {
+		if res.Labels != o.Selector[0] {
+			return errors.New("No match found")
+		}
+	} else if len(o.Selector) >= 2 {
+
+		for _, val := range o.Selector {
+			if val == res.Labels {
+				return nil
+			}
+		}
+
+	}
+	return errors.New("Not found any flag")
+}
+
+func selectLabelsLogs(o Options, res *pb.Log) error {
+
+	if len(o.Selector) <= 1 {
```

Comment:
Not sure about why we need to explicitly add a check for this. Check the proposed code for selectLabelsAlerts